pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
> For now, what i will do is remove the doc changes, and will submit a PR or not based on what the other teams use and what i find out. I'm guessing that would ease the work of reviewing this PR, 

sounds good, thank you!

> i'm not trying to sabotage myself :-)

no, just sabotaging @mfojtik and me by editing your comments :)